vars/artifactSetVersion.groovy

```diff
@@ -91,9 +91,9 @@ def call(Map parameters = [:], Closure body = null) {
                 } catch (e) {
                     error "[${STEP_NAME}]git commit failed: ${e}"
                 }
-                sh "git remote set-url origin ${config.gitSshUrl}"
-                sh "git tag ${config.tagPrefix}${newVersion}"
-                sh "git push origin ${config.tagPrefix}${newVersion}"
+                sh """#!/bin/bash
+                      git tag ${config.tagPrefix}${newVersion} && \\
```

Comment:
no need tor `&&` in multi line shell calls

Reply:
you are right.